### ISSUE TITLE
Add shell completion of container and image names

### ISSUE THREAD

**Issue by swilmet-uclouvain** [containers/buildah#1197]
Add shell (bash) completion of container and image names.

When I type a command like:
```
# buildah rm
```
I would like to be able to complete the name of containers.

The same when an image name needs to be given.

**Comment by rhatdan** [comment]
Works for me?

# buildah rm <tab><tab>
3df3377f101af284557ae528c9f38e7d69f4fb8055714a9845811d30cc82db96
8f0455b0d799602cf54445a20212cbc2de932069f6510f47f5417eecccb9bb9a

Seems to work for me.

```
buildah containers -q
3df3377f101af284557ae528c9f38e7d69f4fb8055714a9845811d30cc82db96
8f0455b0d799602cf54445a20212cbc2de932069f6510f47f5417eecccb9bb9a
```
 rpm -q buildah
buildah-1.4-3.git608fa84.fc29.x86_64

**Comment by swilmet-uclouvain** [comment]
Ah ok yes, for `buildah rm` and `buildah rmi` it completes the IDs. What I was trying to do is to start writing the container/image *name* and then press Tab, but it doesn't complete the names, just the IDs.

A better example is with `buildah run`, when we want to provide the container name. It's easy to remember the name, not the ID, so the completion of the container name would be useful for that command.

**Comment by rhatdan** [comment]
Fixed in master.  Fix will be in buildah-1.6.

**Comment by swilmet-uclouvain** [comment]
Thanks!
